packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -311,7 +311,10 @@ export class LitContracts {
     // ----------------------------------------------
     else if (isNode()) {
       this.log("----- We're in node! -----");
-      this.provider = new ethers.providers.StaticJsonRpcProvider(this.rpc);
+      this.provider = new ethers.providers.StaticJsonRpcProvider({
+        url: this.rpc,
+        skipFetchSetup: true,
```

Comment:
Do we have a PR or branch with this @MaximusHaximus ? If we are not accepting the provider object we could accept provider configs that override our defaults such as this one

Reply:
Ethers sets the fetch setup values to their existing _default values_, which NextJS doesn't like -- but in other environments, the change is a no-op because the values _are the defaults_  -- so we shouldn't need to expose that we're setting this flag on the provider at all.

IMO, let's go ahead and land this fix in v6 without forcing users to upgrade to v7 to get this fix, even if we **_do_** decide to make this part of the NetworkConfig object in v7.

Reply:
This PR is already merging to master which is v6 already?

Reply:
Yup!